pipelines/firebase/__init__.py

```diff
@@ -0,0 +1,84 @@
+"""Loads data from firebase realtime database"""
+
+from typing import Any, Iterator, Sequence, Union
+
+import dlt
+from dlt.extract.source import DltResource
+from dlt.common.typing import TDataItem
+from dlt.sources.credentials import GcpOAuthCredentials, GcpServiceAccountCredentials
+from dlt.common.exceptions import MissingDependencyException
+
+try:
+    from firebase_admin import credentials, db
+except ImportError:
+    raise MissingDependencyException("Firebase Admin Python SDK", ["firebase-admin"])
+
+
+_DATA =  {
```

Comment:
this is a demo  case. please move it to demo script. best way would be to create another inline source in firebase_pipleine.py that just creates the app and fixture and then call it (like you call this one). please make this pipeline generic